app/models/account/trade_builder.rb

```diff
@@ -31,6 +31,7 @@ def create_entry
     end
 
     def security
+      return Security.find(ticker) if ticker.match?(/\A[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}\z/i)
       Security.find_or_create_by(ticker: ticker)
```

Comment:
Does this second `find_or_create_by` line get hit anymore now that we have the combobox?

Reply:
Yes. You can still type whatever you want in the box in the event we don't have support for a given ticker.

I guess there's an argument to be made to only allow a ticker we support, but I do think that reduces some of the flexbility of the account type.